.github/actions/create-pr/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Create Pull Request
+gh pr create --title "Release ${VERSION}" --body "Auto-commit"
```

Comment:
[nit] is body the description of the PR? Maybe we can put a bit more information here?

Reply:
where does `${VERSION}` come from? Don't see it in the `inputs:` in `action.yml`

Reply:
it is environment variable